src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/AsyncHelpers.Browser.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace System.Runtime.CompilerServices
+{
+    public static partial class AsyncHelpers
+    {
+        [DynamicDependency(nameof(MainWrapper))]
+        [DynamicDependency(nameof(MainWrapperVoid))]
+        static AsyncHelpers()
```

Comment:
Empty explicit static constructor adds overhead to all static methods on the type. Why is this needed?

Reply:
Just quick hack to have something that can have `DynamicDependency` on it. 
It could be moved on any other SPC method that survives IL trimming on browser. Do you have a preference ?

Reply:
Those managed methods would be IL trimmed and then not found when I want to call them from C. See `METHOD__ASYNC_HELPERS__MAIN_WRAPPER` on this PR